content/container-environment.md

```diff
@@ -12,13 +12,15 @@ Galaxy runs your app in a set of containers using the [Docker](https://www.docke
 
 <h2 id="network">Network environment</h2>
 
-Galaxy runs your app containers in a firewalled network environment.  Only one port is exposed for external connections.  Galaxy tells your app what port to listen on via the `$PORT` environment variable, which contains a number such as `3000`.
+Galaxy runs your app containers in a firewalled network environment.  Only one port is exposed for external connections.  Galaxy tells your app what port to listen on via the `$PORT` environment variable, which contains a number such as `3000`. You can set this number by setting the `$PORT` environment variable.
```

Comment:
No, Galaxy assigns this and you read it, not the other way around.  I think technically you can override the $PORT env var but then your thing just won't work. This will be more clear once logger2 is fully out and there is a brief list of Galaxy-generated environment variables.